ui/src/app/app.component.css

```diff
@@ -2,17 +2,19 @@ main {
      height: 100vh;
      overflow: hidden;
 }
+
 app-login-screen {
```

Comment:
It's not typically advisable to target custom Angular tags with CSS; the behaviour isn't consistent from browser-to-browser.